includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -57,6 +66,15 @@ public function register_routes() {
 				],
 			]
 		);
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base . '/(?P<order_id>\w+)/create_customer',
```

Comment:
Nit: Maybe we'd like to ensure `order_id` is an integer?
```suggestion
			$this->rest_base . '/(?P<order_id>\d+)/create_customer',
```

Reply:
Sure.

I tried to see the difference response when non-numeric is passed as order id. They both respond with `404 Not Found`, only the body that is different, which is OK.

No route:
```json
{"code":"rest_no_route","message":"No route was found matching the URL and request method.","data":{"status":404}}
```
vs
No order found:
```json
{"code":"wcpay_missing_order","message":"Order not found","data":{"status":404}}
```